po/ko.po

```diff
@@ -920,7 +920,7 @@ msgstr ""
 #: src/android/interoperability/with-c/rust.md:81
 #: src/exercises/day-4/afternoon.md:10
 msgid "<details>"
-msgstr ""
+msgstr "<details>"
```

Comment:
Just a tiny comment: you're completely correct that you can leave it as `""`. However, it's better if you do translate this (even if it's a no-op): this way the string no longer counts as an untranslated string. The same applies to the code snippets: almost all of them don't require translation, so they should just be copied into the field for Korean.

Reply:
Yes, but at the same time we will have to copy-and-paste the non-translatable strings everytime they are updated in the source.

It will be nice if we have a mechanism for annotating a block of paragraphs as non-translatable.

Reply:
Yeah, we can do better when extracting strings from the Markdown files. I added some ideas for improvements to #318.